content/en/dashboards/widgets/wildcard.md

```diff
@@ -41,6 +42,22 @@ However, if none of the Datadog widgets meets your visualization needs, a Wildca
 1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
 1. Click **Save**.
 
+#### Formulas Scalar vs. Formulas Timeseries
+
+In Datadog dashboards, scalar and timeseries visualizations serve different purposes and are used to represent data in distinct ways.
+
+**Timeseries**
+: This visualization is designed to display how your data changes over time.
+   - **Use-cases**: It's ideal for monitoring metrics that fluctuate, such as CPU usage, memory consumption, or request rates. It helps identify trends, patterns, and anomalies over a specified time range. 
+   - **Appearance**: These visualizations are typically line or area charts. They show datapoints plotted against time (usually on the x-axis), to observe changes and trends.
+
+**Scalar**
+: This visualization presents a single, aggregate value derived from your data.
+   - **Use-cases**: It's best for displaying key performance indicators (KPIs) or summary statistics such as averages, sums, or percentiles. It provides a summary view of the current state or a specific metric.
+   - **Appearance**: Scalars often appear as large, standalone numbers or small cards on dashboards, highlighting the most relevant statistics, like the current number of active users, or the sum of errors over the last hour.
+
+Timeseries visualizations emphasize data trends over time, scalar visualizations focus on presenting single, computed values that help with quick assessments. If you're going to visualize time on an axis, or need individual time buckets, choose the Timeseries type. However, if you're not vizualizing against time, choose the Scalar type as it's much more performant.
```

Comment:
```suggestion
The Timeseries data format emphasizes data trends over time, while the Scalar format focuses on presenting single, computed values for quick assessments. Choose the Timeseries type if you need to visualize time on an axis or require individual time buckets. If not visualizing against time, select the Scalar type for performance. 

**Note**: The "Formulas" prefix is used specifically for Scalar and Timeseries formats because they are compatible with the Functions API. The other formats, such as Histogram and List, do not support this API.
```